test/sandbox/sandbox.go

```diff
@@ -152,6 +152,8 @@ func (s S) BuildTree(layout []string) {
 				cfg.Stack.After = specList(t, name, value)
 			case "before":
 				cfg.Stack.Before = specList(t, name, value)
+			case "wants":
+				cfg.Stack.Wants = specList(t, name, value)
```

Comment:
Unrelated to the PR, but thinknig that maybe specList should be parseListSpec ? (or parseStrList ?)